server/frontend/src/components/ProviderKey.vue

```diff
@@ -156,4 +156,7 @@ export default defineComponent({
 .mt-light {
   margin-top: 0.5rem;
 }
+.btn {
+  margin-left: 0.8rem;
+}
```

Comment:
Unrelated, but fixes existing  issue where there is no gap between the button and input here:

<img width="313" alt="Screenshot 2024-12-27 at 6 46 54 am" src="https://github.com/user-attachments/assets/7cfb5fcc-8650-4142-bfbb-13d37290fa87" />

Reply:
Thank you!